package.json

```diff
@@ -20,6 +20,7 @@
   },
   "homepage": "https://github.com/memgraph/jsmgclient#readme",
   "dependencies": {
+    "js-beautify": "^1.14.0",
```

Comment:
Add this as a dev dependency (https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file)

Reply:
you learn every day ;) Cheers!